README.markdown

```diff
@@ -248,6 +248,8 @@ protoc --plugin=node_modules/ts-proto/protoc-gen-ts_proto ./batching.proto -I.
 
 - With `--ts_proto_opt=outputClientImpl=false`, the client implementations, i.e. `FooServiceClientImpl`, that implement the client-side (currently Twirp-only) RPC interfaces will not be output.
 
+- With `--ts_proto_opt=outputJsonClientImpl=true`, an additional client implementation will be output for each service, i.e. `FooServiceJsonClientImpl`, that uses JSON serialization. Has no effect if `outputClientImpl=false`.
```

Comment:
What do you think about doing something like `outputClientImpl = false | true | twirp | json`? I.e. false is off, true is a legacy value that means twirp, and json would be your new code?

I'm kinda thinking that shops will generally have a single preferred client? 

I dunno, maybe not I guess?

Maybe this is better.

Maybe `outputJsonClient=true/false`? I might deprecate the existing `outputClientImpl` and rename it to `outputTwirpClient`...